bindings/matrix-sdk-crypto-js/src/requests.rs

```diff
@@ -33,11 +36,8 @@ pub struct KeysUploadRequest {
     #[wasm_bindgen(readonly)]
     pub id: Option<JsString>,
 
-    /// A JSON-encoded object of form:
-    ///
-    /// ```json
-    /// {"device_keys": …, "one_time_keys": …, "fallback_keys": …}
-    /// ```
+    /// A JSON-encoded string containing the rest of the payload: `device_keys`,
+    /// `one_time_keys`, `fallback_keys`.
```

Comment:
I think it would be good to call out that this is indeed the HTTP body of the request. But I guess that can be inferred from the field name as well.

Reply:
:+1: